package.json

```diff
@@ -5,6 +5,9 @@
   "main": "index.js",
   "type": "commonjs",
   "types": "types/index.d.ts",
+  "bin": {
+    "hash-gen": "./bin/hash-gen.js"
```

Comment:
This seems like a pretty general name for a pretty specific utility from this module that will end up on a lot of user's `PATH`s. What do you think about something like `fastify-static-prehash` or something clunkier like that which would explain where it comes from more when a dev sees it in a script?